### PR TITLE
Fix PEs config and env for ascent

### PR DESCRIPTION
**Pull request by jgfouca**

### CODE REVIEW

File: cime_config/allactive/config_pesall.xml
Comment (jgfouca) [inline cime_config/allactive/config_pesall.xml:1405]: The previous string was not matching our ne30_ne30 grid, which many e3sm_scream_v1 tests use. The long name for that grid is `a%ne30np4_l%ne30np4_oi%ne30np4_r%null_g%null_w%null_z%null_m%gx1v6`. I also had to bump node count for that grid from 2 to 4 as you can see below in order to avoid the memory error from YAKL.

File: cime_config/machines/config_machines.xml
Comment (jgfouca) [inline cime_config/machines/config_machines.xml:3795]: Change env to exactly match summit.

**Comment by jgfouca** [comment]
@PeterCaldwell , you can pass this review along to anyone. I wasn't sure who to add.

**Comment by jgfouca** [comment]
With these changes, I see these results for e3sm_scream_v1 suite:

```
FAIL ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1.ascent_gnugpu (phase COMPARE_base_rest)
    Case dir: /gpfs/wolf/cli115/proj-shared/jgfouca/e3sm_scratch/ERP_D_Ln9.ne4_ne4.F2010-SCREAMv1.ascent_gnugpu.20220803_175933_ntiqgi
PASS ERS_Ln22.ne30_ne30.F2010-SCREAMv1.ascent_gnugpu RUN
    Case dir: /gpfs/wolf/cli115/proj-shared/jgfouca/e3sm_scratch/ERS_Ln22.ne30_ne30.F2010-SCREAMv1.ascent_gnugpu.20220803_175933_ntiqgi
PASS ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.ascent_gnugpu RUN
    Case dir: /gpfs/wolf/cli115/proj-shared/jgfouca/e3sm_scratch/ERS_Ln9.ne4_ne4.F2000-SCREAMv1-AQP1.ascent_gnugpu.20220803_175933_ntiqgi
FAIL PEM_Ln9.ne30_ne30.F2010-SCREAMv1.ascent_gnugpu (phase RUN)
    Case dir: /gpfs/wolf/cli115/proj-shared/jgfouca/e3sm_scratch/PEM_Ln9.ne30_ne30.F2010-SCREAMv1.ascent_gnugpu.20220803_175933_ntiqgi
PASS SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.ascent_gnugpu RUN
    Case dir: /gpfs/wolf/cli115/proj-shared/jgfouca/e3sm_scratch/SMS_D_Ln9.ne4_ne4.F2010-SCREAMv1-noAero.ascent_gnugpu.20220803_175933_ntiqgi
```

So still a couple FAILs but I think we were already struggling with these tests and so not a problem specific to ascent. My branch was fairly out of date when I did this test run and am trying again after an upstream merge.

**Comment by ambrad** [comment]
Great! The ERP fail is consistent with my growing belief that v1 currently has a GPU-related PE-variance issue unrelated to SL/pg2 (which is where I spotted it first).

However, alas for you, I think these changes should go in the upstream repo.

**Review by ambrad — CHANGES_REQUESTED** [review]
As I wrote in a comment, I think this PR needs to go in the upstream repo.

**Comment by jgfouca** [comment]
@ambrad , ok, I will reopen upstream. Thanks for looking at this.

**Comment by PeterCaldwell** [comment]
@jgfouca - Test results look exactly like I was expecting. I think this PR is doing all that we want it to. I would approve it if it hadn't already closed ;-).

**Pull request closed without merging**
